src/icons/index.js

```diff
@@ -0,0 +1,7 @@
+import Info from './Info';
+
+const icons = {
+  info: Info,
```

Comment:
We can rather do this ```  info: require('./Info').default```

Reply:
LMK which is the better approach

Reply:
let's stick with this. Bundlers understand and are able to treeshake this better

Reply:
https://facebook.github.io/react-native/docs/performance#ram-bundles--inline-requires

Check this out. If the list of icon grows, importing this icon file could be expensive since it imports all the icon components at the top level. If we do inline requires, it won't import everything.

Reply:
This is totally opposite of how other bundlers work 🙈 sure then let's do inline